server/server.go

```diff
@@ -52,6 +53,7 @@ func New(c *Config) (*Server, error) {
 	clientCreator, err := githubapp.NewDefaultCachingClientCreator(
 		c.Github,
 		githubapp.WithClientUserAgent(userAgent),
+		githubapp.WithClientCaching(true, func() httpcache.Cache { return httpcache.NewMemoryCache() }),
```

Comment:
Actually, thinking about this more, I believe we want to use the LRU implementation from start. Bulldozer's set of working objects tends to be new/active PRs and a tail of things that are waiting for review or being updated. Once a PR a merged, the application doesn't care about any state related to that PR, but all those object will stick around in the cache forever.

That means it's guaranteed that the cache will grow without bound, eventually using all memory, unless the application is restarted regularly.